src/sg_general.rs

```diff
@@ -396,14 +390,14 @@ pub(crate) fn append_macro_body(
                     #tokens #punct
                 }
             }) {
-                if block.stmts.len() == 1 && match block.stmts.iter().next() {
-                    Some(Stmt::Item(Item::Verbatim(_))) => true,
-                    Some(Stmt::Expr(Expr::Verbatim(_))) => true,
-                    Some(Stmt::Semi(Expr::Verbatim(_), _)) => true,
-                    _ => false,
-                } { 
+                if block.stmts.len() == 1 &&
+                    matches!(
```

Comment:
Off topic, but did clippy warn about this? I added clippy to the CI but maybe it needs some flags tweaked.

Reply:
I only fixed normal clippy lints, nothing special, happy to revet/keep an eye on.

Reply:
Ah yeah, I didn't mean that as a complaint.  Clippy seems reasonable to have more consistent code... The github actions clippy isn't saying anything though so I was wondering if I was missing settings maybe.

Reply:
All good! I just trust clippy basically blindly, the lint suggestions are usually awesome, and if they're not, a simple #[allow] sorts that out. I don't use clippy with any flags, I wouldn't worry too much about it anyway.